packages/i18nify-js/README.md

```diff
@@ -589,3 +589,245 @@ Behold, an object where each key is a country code linked to its flag's URL, suc
 }
 */
 ```
+
+### Module 04: Date & Time Module
+
+This module provides functions for formatting and manipulating dates and times in a locale-sensitive manner using the JavaScript Intl API & Date object.
+
+#### formatDateTime(date, options)
+
+🛠️ Dive into the sophistication of formatDateTime, a highly configurable function designed for developers seeking to master the intricacies of international date and time formatting. Leveraging the robust capabilities of the Intl.DateTimeFormat API, this utility offers unparalleled precision and adaptability in crafting date-time strings. Whether your application caters to a global audience or requires meticulous timestamping, formatDateTime delivers the versatility and accuracy essential for modern software development. 🌍🔧
+
+##### Examples
+
+```javascript
+// Comprehensive date-time formatting with custom options
+console.log(
+  formatDateTime('2024-12-31 23:59', {
+    locale: 'en-US',
+    dateTimeMode: 'dateTime',
+    intlOptions: {
+      weekday: 'long',
+      year: 'numeric',
```

Comment:
I think except for weekday and hour12, all other keys are not required to showcase dateTime